compiler+runtime/src/cpp/jank/read/lex.cpp

```diff
@@ -406,6 +427,30 @@ namespace jank::read
                 }
                 found_exponent_sign = true;
               }
+
+              else if(c == '/') {
+                require_space = false;
+                ++pos;
+                if(found_exponent_sign || is_scientific || expecting_exponent || contains_dot
+                  || found_slash_after_number) {
+                  return err(error{ token_start, pos, "invalid ratio" });
+                }
+                found_slash_after_number = true;
+                /* skip the '/' char and look for the denominator number. */
+                ++pos;
+                auto const denominator(next());
+                if (denominator.is_ok() && denominator.expect_ok().kind == token_kind::integer) {
+                  const token& denominator_token = denominator.expect_ok();
```

Comment:
Please use `auto const &denominator_token(denominator.expect_ok());`

1. Prefer auto
2. Prefer direct initialization
3. const always goes to the right

Reply:
done